my_list = []

try:
    print(my_list[1])
except:
    print("Out of bounds exception occurred")
